### PR TITLE
[Term Entry] Python String Function: format_map()

### PR DESCRIPTION
**Pull request by phuongtrieu97coder**
### Description

-Added the .format_map() term for Python
-Closes issue #3273 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/python/concepts/strings/terms/format-map/format-map.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+Title: '.format_map()'
+Description: 'Returns a dictionary key’s value.'
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/format-map/format-map.md:3]:
```suggestion
Description: 'Returns the values from a given dictionary.'
```

Comment (caupolicandiaz) [reply content/python/concepts/strings/terms/format-map/format-map.md:3]: apostrophes are not permitted in the metadata

File: content/python/concepts/strings/terms/format-map/format-map.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  - 'Data Science'
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/analyze-data-with-python'
+  - 'paths/data-science'
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/format-map/format-map.md:14]:
```suggestion
```

Comment (caupolicandiaz) [reply content/python/concepts/strings/terms/format-map/format-map.md:14]: there should only be 2 items under catalog content

File: content/python/concepts/strings/terms/format-map/format-map.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+---
+
+The `.format_map()` string method takes a single parameter which is the input dictionary and returns key's values.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/format-map/format-map.md:17]:
```suggestion
The **`.format_map()`** string method takes a single parameter which is the input dictionary and returns the key's values.
```

File: content/python/concepts/strings/terms/format-map/format-map.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+## Syntax
+
+```py
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/format-map/format-map.md:21]:
```suggestion
```pseudo
```

File: content/python/concepts/strings/terms/format-map/format-map.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+string.format_map(d)
+```
+
+`d` is a variable in which the input dictionary is stored.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/format-map/format-map.md:25]:
```suggestion
`d` is the dictionary to be addressed.
```

File: content/python/concepts/strings/terms/format-map/format-map.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+## Example
+
+The example below uses `.format_map()` takes in a variable `a` which stores a dictionary and returns the value of the key `x` and `y`:
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/format-map/format-map.md:29]:
```suggestion
The example below uses `.format_map()` to return the values stored in `x` and `y`:
```

File: content/python/concepts/strings/terms/format-map/format-map.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+```py
+# input stored in variable a.
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/format-map/format-map.md:32]:
```suggestion
# Input stored in variable a.
```

File: content/python/concepts/strings/terms/format-map/format-map.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+```
+
+The output will look like this:
+
+```shell
+That DJ is Mike Perry
+```
+
+## Codebyte Example
+
```
Comment (caupolicandiaz) [inline content/python/concepts/strings/terms/format-map/format-map.md:46]:
```suggestion
The following code is runnable and demonstrates the use of `.format_map()`:
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@phuongtrieu97coder I've added my initial comments. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by phuongtrieu97coder** [comment]
@caupolicandiaz I've completed my review and finished making changes.

**Review by caupolicandiaz — APPROVED** [review]
@phuongtrieu97coder I just made a couple of spacing edits directly. This is ready for a second review.

**Review by SSwiniarski — APPROVED** [review]
@phuongtrieu97coder @caupolicandiaz Looks good.

**Pull request merged**